inventoryLocalStore/index.js

```diff
@@ -14,6 +15,29 @@ class Inventory {
       .then(result => result.rows)
       .catch(console.error);
   }
+
+  getAvailableListings(market, start, end, limit) {
+    const queryLimit = limit ? ` FETCH FIRST ${limit * getLengthOfStay(start, end)} ROWS ONLY` : '';
+    const queryString = `
+      SELECT DISTINCT *
+      FROM listings l
```

Comment:
Clarify what l is or make it more descriptive.

Reply:
OK, I'll add the "AS" syntax to make the aliasing more clear.